tmt/convert.py

```diff
@@ -325,6 +325,15 @@ def read(path, makefile, nitrate, purpose, disabled):
             echo(
                 style('recommend: ', fg='green') + ' '.join(data['recommend']))
 
+        # Multihost (from Type) -> Add tag for now
+        try:
+            type = re.search(
+                r'^Type:\s*(.*)', testinfo, re.M).group(1)
```

Comment:
isn't better here directly match `re.match(`Type:.*Multihost', testinfo, re.M)`
because I do not know these markers well. if possible e.g. to write `Type: Sanity Multihost` os something similar more complex?

Reply:
Good point. I thought only one value was allowed but looks that it isn't the case